packages/apostrophe/modules/@apostrophecms/schema/lib/newInstance.js

```diff
@@ -99,7 +99,7 @@ function normalizeWidget(widgetInstance, self) {
     }
   }
 
-  // Contextual default data
+  // Contextual default data other than schema fields
```

Comment:
This is just for clarity. This file ALREADY instantiates schema fields for contextual widgets, but this particular bit of code is about the wacky weirdies like `content`.